<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>boxeditorClass</class>
 <widget class="QMainWindow" name="boxeditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>572</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>boxeditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTableWidget" name="tblWords">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>Word</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Left</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Bottom</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Right</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Top</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Page</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="zoom">
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="currentText">
             <string>100%</string>
            </property>
            <item>
             <property name="text">
              <string>20%</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>50%</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>75%</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>100%</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>150%</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>200%</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>300%</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>400%</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>500%</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>600%</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>700%</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>800%</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="img_view">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>682</width>
            <height>509</height>
           </rect>
          </property>
          <widget class="QLabel" name="box">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>100</y>
             <width>61</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1090</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenFolder"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menuImages">
    <property name="title">
     <string>Images</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuImages"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open Image</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionOpenFolder">
   <property name="text">
    <string>Open Folder...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="boxeditor.qrc"/>
 </resources>
 <connections/>
</ui>
